curation_concerns-models/app/models/concerns/curation_concerns/has_representative.rb

```diff
@@ -2,12 +2,8 @@ module CurationConcerns::HasRepresentative
   extend ActiveSupport::Concern
 
   included do
-    property :representative, predicate: RDF::URI.new('http://opaquenamespace.org/ns/hydra/representative'), multiple: false
-  end
-
-  def to_solr(solr_doc = {})
-    super.tap do |doc|
-      doc[Solrizer.solr_name('representative', :stored_searchable)] = representative
-    end
+    belongs_to :representative,
+               predicate: ::RDF::URI('http://opaquenamespace.org/ns/hydra/representative'),
```

Comment:
I couldn't find a more appropriate predicate.  I have an email in to our metadata librarian, but updating the predicate could always be a subsequent issue.